classes/Bili/Date.php

```diff
@@ -153,18 +180,18 @@ public static function parseDate($strDate, $strFormat)
      * @param string $strFormat
      * @param int $intMinYear
      * @param int|null $intMaxYear
-     * @return \DateTime|null
+     * @return DateTime|null
      */
-    public static function testParsedDate($strDate, $strFormat, $intMinYear, $intMaxYear = null)
+    public static function testParsedDate($strDate, $strFormat, $intMinYear, $intMaxYear = null): ?DateTime
```

Comment:
Please add PHP types to the input parameters and rename `$strFormat` to `strIsoFormat`. Also adjust the doc block.